Sources/Anthropic/Public/Parameters/Message/MessageParameter.swift

```diff
@@ -211,6 +213,22 @@ public struct MessageParameter: Encodable {
       // This should be a uuid, hash value, or other opaque identifier. Anthropic may use this id to help detect abuse. Do not include any identifying information such as name, email address, or phone number.
       public let userId: UUID
    }
+
+   public struct ToolChoice: Codable {
+      public enum ToolType: String, Codable {
+         case tool
+         case auto
+         case any
+      }
+      
+      let type: ToolType
+      let name: String
```

Comment:
The `name` parameter is only used when `ToolType` is set to `tool`. Shouldn’t we prevent it from being specified when ToolType is set to `auto` or `any` ?

Reply:
Thanks for flagging, will take care of this in a follow up PR